windows_containers/wmco_rn/windows-containers-release-notes-10-17-x-prereqs.adoc

```diff
@@ -4,7 +4,14 @@
 = Windows Machine Config Operator prerequisites
 include::_attributes/common-attributes.adoc[]
 
-The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
+The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator (WMCO). See the vSphere documentation for any information that is relevant to only that platform.
+
+[id="wmco-prerequisites-supported-install-10.17.0_{context}"]
+== WMCO supported Installation method
+
+The WMCO supports installing Windows nodes into {product-title} clusters installed by using the Installer-Provisioned Infrastructure (IPI) installation method. This is the preferred installation method.
+
+For the User-Provisioned Infrastructure (UPI) installation method, the WMCO supports installing Windows nodes into {product-title} clusters installed for only bare-metal or provider-agnostic clusters, and only for the xref:../../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[BYOH (Bring Your Own Host)] use case. UPI is not supported for any other platform.
```

Comment:
bare-metal can be misleading, customer has to configure platform=none and NOT platform=baremetal